src/components/blipModal/index.js

```diff
@@ -0,0 +1,87 @@
+import { ModalContainer } from './ModalContainer'
+
+export const MODAL_DEFAULTS = {
+  onOpen: () => {},
+  onClose: () => {},
+  onOk: () => {},
+  onCancel: () => {},
+  closeOnOverlayClick: true,
+  closeOnOk: true,
+  closeOnCancel: true,
+  okButton: { text: 'Ok', classes: '' },
```

Comment:
Same of above, isn't it better "confirm" instead of "ok" (in all cases)?

Reply:
Done!